content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md

```diff
@@ -0,0 +1,147 @@
+
+![01-Grafana-dashboard](/images/blogs/en/release-announcement3.2.0/01-Grafana-dashboard.png)
+
+For GPU resources, KubeSphere 3.2.0 also provides a default monitoring template with a wealth of metrics, so that you don't need to customize a template or edit a YAML file.
+
+![02-GPU-overview](/images/blogs/en/release-announcement3.2.0/02-GPU-overview.png)
+
+2. Alerting and logging
+
+- KubeSphere 3.2.0 supports communication with Elasticsearch through HTTPS.
+
+- In addition to the various notification channels such as email, DingTalk, WeCom, webhook, and Slack, KubeSphere 3.2.0 now also allows you to test and validate the notification channels you configure.
+
+![03-platform-settings](/images/blogs/en/release-announcement3.2.0/03-platform-settings.png)
+
+3. On the etcd monitoring page, the system automatically adds the `Leader` tag to the etcd leader.
+
+### **Multi-cloud and multi-cluster management**
+
+CNCF Survey 2020 shows that over 80% of users run more than two Kubernetes clusters in their production environment. KubeSphere aims at addressing multi-cluster and multi-cloud challenges. It provides a unified control plane and supports distributing applications and replicas to multiple clusters deployed across public cloud and on-premises environments. Moreover, KubeSphere supports observability across clusters, including features such as monitoring, logs, events, and auditing logs of multiple dimensions.
+
+![04-cluster-manage](/images/blogs/en/release-announcement3.2.0/04-cluster-manage.png)
+
+KubeSphere 3.2.0 performs better in cross-cluster scheduling. When you are creating a federated Deployment across clusters, you can directly specify the number of replicas scheduled to each cluster. In addition, you can also specify the total number of replicas and weight of each cluster, and allow the system to automatically schedule replicas to each cluster according its weight. This feature is pretty helpful when you want to flexibly scale your Deployment and proportionally distribute replicas to multiple clusters.
+
+![05-federated-deployment](/images/blogs/en/release-announcement3.2.0/05-federated-deployment.png)
+
+![06-view-federation](/images/blogs/en/release-announcement3.2.0/06-view-federation.png)
+
+### **Operations and maintenance-friendly storage management**
+
+Persistent storage is the most important focus for enterprises running Kubernetes in their production environment, and stable and reliable storage guarantees the safety of core enterprise data. On the KubeSphere 3.2.0 web console, the **Volumes** feature allows the administrator to decide whether to enable volume cloning, snapshot capturing, and volume expansion, making persistent storage operations and maintenance for stateful apps more convenient.
```

Comment:
Is it necessary to keep `,and` between `environment` and `stable`? Shall we divide it into two individual sentences?

Reply:
fixed